tests/actions.go

```diff
@@ -1014,7 +1014,7 @@ func (oa *operatorActions) CheckTidbClusterStatus(info *TidbClusterConfig) error
 	ns := info.Namespace
 	tcName := info.ClusterName
 	// TODO: remove redundant checks already in WaitForTidbClusterReady
-	if err := wait.Poll(oa.pollInterval, 10*time.Minute, func() (bool, error) {
+	if err := wait.Poll(oa.pollInterval, 20*time.Minute, func() (bool, error) {
```

Comment:
The default evictLeaderTimeOut value has been changed to 10 Minutes, however, we cannot configure the value for the clusters deployed with Helm chart, so extend the check duration here.